src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = !century
```

Comment:
Maybe it's just me but try to think how another programmer will read your code and understand your variables.
For example `filterMen`, at first sight, I thought it should be a callback. Perhaps `filteredMen` or `onlyMen` would be better. 🤗